lib/refile/app.rb

```diff
@@ -85,6 +85,16 @@ class App < Sinatra::Base
       "forbidden"
     end
 
+    error ArgumentError do
```

Comment:
ArgumentError is a very general error to rescue. I think it's not ideal to assume that any particular thing happened given an ArgumentError.